service/static/js-src/muk.institution.js

```diff
@@ -4219,7 +4228,7 @@ $.extend(muk, {
             var inst_buckets = ch.edge.result.buckets("institution");
             for (var i = 0; i < inst_buckets.length; i++) {
                 if (maxSeries > 0 && i >= maxSeries) {
-                    break
+                    return []
```

Comment:
We don't want for this method to generate the default horizontal bar values (at this point - Loading time or no options selected). And because we have to go through this point at this time we have to return an empty array.